spark/common/pom.xml

```diff
@@ -122,6 +122,10 @@
             <groupId>org.geotools</groupId>
             <artifactId>gt-geotiff</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-process-feature</artifactId>
```

Comment:
I think we need to add it to geotools-wrapper. I'm afraid adding it to spark-shaded will have any licensing issues.

Reply:
It reminds me that we also need to add this to geotools-wrapper.

Reply:
@Kontinuation The geotools scope in spark-shaded is `provided` so we should be fine. Putting it in `spark-shaded` will help Python and R tests in the future since they can simply do `mvn clean install -Dgeotools`